types/plugin/timezone.d.ts

```diff
@@ -11,6 +11,7 @@ declare module 'dayjs' {
   interface DayjsTimezone {
     (date: ConfigType, timezone: string): Dayjs
     guess(): string
+    setDefault(timezone?: string): Dayjs
```

Comment:
setDefault function returns nothing

Reply:
I got it, I'm replacing `Dayjs` to `void`, Thanks!